dash-pipeline/SAI/sai_api_gen.py

```diff
@@ -118,6 +120,11 @@ def get_sai_key_data(key):
     else:
         raise ValueError(f"match_type={sai_key_data['match_type']} is not supported")
 
+    if 'sai_key_type' in sai_key_data and  sai_key_data['sai_key_type'] == 'sai_object_id_t':
+        if sai_key_data['sai_key_name'].endswith('dash_acl_group_id'):
```

Comment:
Why is there an explicit reference to a table from DASH?

Reply:
Will remove this. My original change had the acl rule key name as acl_group_id. So I had added this to fix the @objects meta in the header to the correct dash table name. But later I changed the acl rule key name to dash_acl_group_id. So its not needed here anymore. 

Now this is only needed for ENI params like SAI_ENI_ATTR_INBOUND_V4_STAGE1_DASH_ACL_GROUP_ID.

Reply:
Will remove this. My original change had the acl rule key name as acl_group_id. So I had added this to fix the @objects meta in the header to the correct dash table name. But later I changed the acl rule key name to dash_acl_group_id. So its not needed here anymore.